python-sdk/nuimages/nuimages.py

```diff
@@ -0,0 +1,342 @@
+# nuScenes dev-kit.
+# Code written by Asha Asvathaman & Holger Caesar, 2020.
+
+import sys
+import os.path as osp
+import json
+import time
+from typing import Any, List, Dict, Optional
+from collections import defaultdict
+
+import PIL
+import PIL.ImageDraw
+import PIL.ImageFont
+import PIL.Image
+import matplotlib.pyplot as plt
+from matplotlib.axes import Axes
+
+from nuimages.utils import default_color, annotation_name, mask_decode
+
+PYTHON_VERSION = sys.version_info[0]
+
+if not PYTHON_VERSION == 3:
+    raise ValueError("nuScenes dev-kit only supports Python version 3.")
```

Comment:
nuImages

Reply:
Same as above. We'll keep calling it nuScenes devkit